### PR TITLE
Handle plugin gracefully

### PR DESCRIPTION
**Pull request by superstar54**
This PR tries to handle plugin gracefully in two ways:

- when the user installs the plugin from the plugin page, after the installation, it will try to load the entry point; if it failed, show the message and uninstall.

https://github.com/aiidalab/aiidalab-qe/assets/11457659/aa62c794-7010-4538-86d1-2791bb5a1f18

- if user install a plugin manually, when run the QEApp, it will show warning message if QEApp can not load the entry point. Here, I created a bug in xas plugin and test:

Screenshot from 2024-05-01 11-15-05

### CODE REVIEW

**Comment by superstar54** [comment]
Hi @AndresOrtegaGuerrero , I added a check for `ComputationalResourcesWidget` explicitly, so that when you install `aiidalab-qe-vibroscopy`, it will show the error.

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM! , I tested both cases you considered and it works perfectly !  Nice!

File: plugin_list.ipynb [hunk 1]
```diff
@@ -97,29 +97,52 @@
...
+    "            # restart daemon\n",
+    "            message_container.value = \"\"\"<div style=\"color: #008000;\">Loading plugin test passed.</div>\"\"\"\n",
+    "            message_container.value += \"\"\"<div style=\"color: #008000;\">Plugin installed successfully.</div>\"\"\"\n",
+    "            accordion.set_title(index, f\"{accordion.get_title(index)[:-2]} ✅\")\n",
+    "            command = [\"verdi\", \"daemon\", \"restart\"]\n",
+    "            subprocess.run(command, capture_output=True, shell=False)\n",
+    "        else:\n",
+    "            # uninstall the package\n",
+    "            message_container.value = f\"\"\"<div style=\"color: #FF0000;\">The plugin '{package_name}' was installed successfully but plugin functionality test failed: {result.stderr}. </div>\"\"\"\n",
+    "            message_container.value += \"\"\"<div style=\"color: #FF0000;\">This may be due to compatibility issues with the current QEApp version. Please contact the plugin author for further assistance.</div>\"\"\"\n",
```
Comment (AndresOrtegaGuerrero) [inline plugin_list.ipynb:124]: current AiiDAlab QEApp ?

Comment (superstar54) [reply plugin_list.ipynb]: Thanks, I added `AiiDAlab`.

**Comment by superstar54** [comment]
@AndresOrtegaGuerrero , thanks for your review!

**Pull request merged**